universal-login-web3/src/ui/react/ULWeb3Root.tsx

```diff
@@ -18,15 +18,16 @@ export const ULWeb3Root = ({sdk, walletService, uiController, domains}: ULWeb3Ro
   const modal = useProperty(uiController.activeModal);
   const message = sdk.getNotice();
 
+  if (uiController.isLoading.get()) {
```

Comment:
This won't update the react component. Use `usePropperty` instead

```suggestion
  if (useProperty(uiController.isLoading)) {
```